docs/kind.md

```diff
@@ -56,6 +56,14 @@ antrea-agent-zsztq                   2/2     Running   0          8m56s
 antrea-controller-775f4d79f8-6tksp   1/1     Running   0          8m56s
 ```
 
+### Short Cut
+ Alternatively to create a two worker node cluster with antrea installed, do
+
+  ./hack/kind-setup.sh create CLUSTER_NAME
```

Comment:
I don't think it's great for users to have to use a modified version of kind (or in this case a script that wroas the kind binary), so I wouldn't advertise this script like this here, and probably move it to `ci/kind/`. While I appreciate the fact that your changes are useful for CI, for a user who just wants to deploy Antrea to a Kind cluster to quickly experiment with network policy support, different networking modes are irrelevant IMO.

Reply:
ok will move to ci/kind